Dockerfile

```
@@ -10,13 +10,22 @@ WORKDIR /opt/app/
 # System deps and get the ohsome-api repository data:
 RUN apk add git bash maven && git clone https://github.com/GIScience/ohsome-api.git ./ && git fetch --all --tags
 
+COPY fallback_data/fallback.tar.xz /opt/app/
+
 # Checkout version if provided
 RUN if [ -z $OHSOMEAPI_VERSION ] || [ "$OHSOMEAPI_VERSION" = "latest" ] ; then echo Version not provided. Sticking to latest version.; else echo Version provided. Checkout $OHSOMEAPI_VERSION \
-    && git checkout --quiet tags/$OHSOMEAPI_VERSION ; fi
+    && git checkout --quiet tags/$OHSOMEAPI_VERSION ; fi \
+    && echo Extract the test data. \
+    && tar -xf fallback.tar.xz \
+    && rm -rf fallback.tar.xz
```

If you apply the aforementioned suggestion:
```suggestion
RUN echo Extract the test data. \
    && tar -xf fallback.tar.xz \
    && rm -rf fallback.tar.xz
```